src/providers/irc/IrcMessageBuilder.hpp

```diff
@@ -36,10 +36,23 @@ class IrcMessageBuilder : public SharedMessageBuilder
     explicit IrcMessageBuilder(const Communi::IrcNoticeMessage *_ircMessage,
                                const MessageParseArgs &_args);
 
+    /**
+     * @brief used for global private/direct messages (i.e. PRIVMSG messages with our nick as the target)
```

Comment:
Let's be consistent with whatever naming we choose for this (whispers, direct messages, or private messages)